Sources/Player/Player.swift

```diff
@@ -360,6 +360,26 @@ public extension Player {
 		}
 	}
 
+	func offline(mediaProduct: MediaProduct) -> Bool {
+		offlineEngine?.offline(mediaProduct: mediaProduct) ?? false
+	}
+
+	func deleteOffline(mediaProduct: MediaProduct) -> Bool {
```

Comment:
[suggestion] if we mean the media product, maybe it reads better
```suggestion
	func deleteOfflined(mediaProduct: MediaProduct) -> Bool {
```
or
```suggestion
	func deleteOfflinedMediaProduct(mediaProduct: MediaProduct) -> Bool {
```
or
```suggestion
	func deleteOfflinedItem(mediaProduct: MediaProduct) -> Bool {
```
or something similar.